### PR TITLE
Remove Visual Studio as a prerequisite

### PR DESCRIPTION
**Pull request by JosephWoodward**
With .NET Core now being cross-platform and IDE agnostic, listing Visual Studio 2017 as a prerequisite (thus *required*) is misleading and confusing to newcomers to the platform using one of the many alternative editors.

As a .NET developer using a Macbook with VS Code, Visual Studio for Mac and/or JetBrains' Rider, this prerequisite implies that I'm unable to upgrade to .NET Core 2.0 because I don't have Visual Studio 2017.

An alternative approach could be to reword it to something similar to 

> Visual Studio 2017, Visual Studio for Mac, Visual Studio Code or an alternative .NET Core compatible IDE

If this is preferred then let me know and I'll update my PR.

When creating a new PR, please do the following and delete this template text:

* Reference the issue number if there is one:

  Fixes #Issue_Number

  > The "Fixes #nnn" syntax in the PR description causes
  > GitHub to automatically close the issue when this PR is merged.

* Suggest reviewers if you know who should review the PR, using the GitHub **Reviewers** UI.

### CODE REVIEW

**Comment by scottaddie** [comment]
@JosephWoodward Thank you for pointing this out! Your change will go live within the next 24 hours.

**Comment by JosephWoodward** [comment]
@scottaddie Not a problem, glad to be of help!
